tailwind.config.js

```diff
@@ -9,6 +9,10 @@ module.exports = {
         primary: '#B70569',
         darkAccent: '#210D15'
       },
+      fontFamily: {
+        header: '"DM Sans"',
```

Comment:
Hey @johnGachihi,
The `header` font class won't be generated because it does not exist in the default Tailwind config.
To add the new `header` font class, it needs to be in `theme.fontFamily` object, outside extend. [Docs references](https://tailwindcss.com/docs/font-family#customizing-your-theme)

Here is what the theme object would look like with the custom fonts:
```javascript
theme: {
    fontFamily: {
      header: ['"DM Sans"', ...defaultTheme.fontFamily.sans],
    },
    extend: {
      colors: {
        primary: '#B70569',
        darkAccent: '#210D15'
      },
      fontFamily: {
        sans: ['Poppins', ...defaultTheme.fontFamily.sans]
      }
    },
  },
```
Tailwind's default theme provides cross-browser font family definitions for the sans stack. It is common to use them as fallback fonts as shown in the snippet.

Reply:
Thanks @steekam

Reply:
Thanks for the tip @steekam.